pkg/platform/openstack/networking.go

```diff
@@ -114,6 +115,21 @@ func findOpenStackNetwork(client *gophercloud.ServiceClient, tag string) (networ
 	}
 }
 
+// Gets the MTU of a Network
+func getOpenStackNetworkMTU(client *gophercloud.ServiceClient, networkID string) (int, error) {
+	networkMTU := 0
+	type NetworkMTU struct {
+		networks.Network
+		mtu.NetworkMTUExt
+	}
```

Comment:
I'm not sure how it will work with golang - but shouldn't that type definition be rather defined outside of the function?

Reply:
The scope of this `struct` is only meant to be for inside the function as that is the only place it's used.I believe there is no need to make it a available at package level.

Reply:
I just hope, the compiled code will not create this struct again, and again while calling this function :)